### PR TITLE
Add Twig sec issue

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: twig/twig/2022-09-28.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title:     Possibility to load a template outside a configured directory when using the filesystem loader
+link:      https://symfony.com/blog/twig-security-release-possibility-to-load-a-template-outside-a-configured-directory-when-using-the-filesystem-loader
+cve:       ~
```
Comment (stof) [inline twig/twig/2022-09-28.yaml:3]: no CVE assigned for it ?

Comment (fabpot) [reply twig/twig/2022-09-28.yaml:3]: Not yet, no.

**Review by stof — APPROVED** [review]

File: twig/twig/2022-09-28.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+branches:
+    1.x:
+        time:     2022-09-28 10:36:08
+        versions: ['>=1.0.0', '<1.44.7']
+    2.x:
+        time:     2022-09-28 10:36:08
+        versions: ['>=2.0.0', '<2.15.13']
```
Comment (stof) [inline twig/twig/2022-09-28.yaml:10]: @fabpot are you sure about this version ? The latest one on Packagist is 2.15.3

Comment (fabpot) [reply twig/twig/2022-09-28.yaml:10]: That's a typo indeed. Fixed in a506dc8

**Pull request merged**